docs/version0.30.md

```diff
@@ -60,6 +61,36 @@ would yield the following in the <tt>ENVINFO</tt> section after the complete lis
 
 This option specifies/unspecifies classes of objects that will be hashed and extended with a hash slot upon object creation or first move. This option may improve performance for applications that frequently hash objects of a certain type. See [-XX:[+|-]EnsureHashed](xxensurehashed.md) for more details.
 
+### Redesigned heap resizing for balanced GC policy
+
+Heap resizing heuristics have been redesigned for balanced GC. This includes both total heap resizing including Eden and non-Eden components indenpendantly, and also balancing between these 2 components when heap is fully expanded. The heuristics now combine both CPU overhead (for both Partial GCs and Global Mark Phase) and heap occupancy criteria. The balancing between Eden and non-Eden for fully expanded heaps is far more dynamic (instead of being mostly fixed to ratio 1:4).
+
+As a consequence there should typically be less need for heap sizing tuning options, most notably for Eden sizing options (-Xmn/mnx/mns). 
```

Comment:
Pls add a link and change the order to be alphabetical
i.e. `[-Xmn/mns/mnx](xmn.md)`